alkira/provider.go

```diff
@@ -83,6 +83,7 @@ func Provider() *schema.Provider {
 			"alkira_service_zscaler":              resourceAlkiraServiceZscaler(),
 			"alkira_service_pan":                  resourceAlkiraServicePan(),
 			"alkira_tenant_network":               resourceAlkiraTenantNetwork(),
+			"alkira_flow_collector":               resourceAlkiraFlowCollector(),
```

Comment:
This list should be in alphabetical order. Could you move it up?

Reply:
done!